.ci/linux-steps.yaml

```diff
@@ -42,9 +42,30 @@ steps:
         sudo make install && \
         cd ..
 
+    # Install ensmallen.
+    wget https://ensmallen.org/files/ensmallen-latest.tar.gz
+        tar -xvzpf ensmallen-latest.tar.gz # Unpack into ensmallen-*/.
+        cd ensmallen-* && \
+        cmake . && \
+        make && \
```

Comment:
```suggestion
```
ensmallen is header-only too, so unless I am forgetting something, I think you can just install directly too. :+1: